### DISCUSSION TITLE
Problems with using Segmented Buttons inside a Panel

### DISCUSSION THREAD

**Discussion by qscheitlin** [UI5/webcomponents-react#4452] [Q&A]
Within an ObjectPageSection, I have a Panel that contains a Toolbar comprising a SegmentedButton.  However, I can't seem to get the SegmentedButtonItem to have a width that fully fits the text. Does anyone know how to solve this? I should also mention moving the SegmentedButton code outside of the Toolbar results in the SegmentedButtonItems having a right width.

<img width="1128" alt="image" src="https://user-images.githubusercontent.com/127273220/229679196-ce19da95-334d-4d16-a01e-16a3dab14b76.png">

Example Code is provided below:

```
<Panel fixed={true}>
              <Toolbar>
                  <SegmentedButton>
                    <SegmentedButtonItem>
                      All ({count.All})
                    </SegmentedButtonItem>
                    <SegmentedButtonItem>
                      Mini-Vans ({count.A})
                    </SegmentedButtonItem>
                    <SegmentedButtonItem>
                      Sedans ({count.B})
                    </SegmentedButtonItem>
                    <SegmentedButtonItem>
                      Automotives ({count.C})
                    </SegmentedButtonItem>
                 </SegmentedButton>
                 <ToolbarSpacer></ToolbarSpacer>
                 <Button icon="add" design="Emphasized">Add Car</Button>
           </Toolbar>
  </Panel>
```

**Comment by Lukas742** [comment]
Converted to Issue #4467
